test/fuzz/TrancheWrapper/TotalAssets.fuzz.t.sol

```diff
@@ -0,0 +1,35 @@
+/**
+ * Created by Pragma Labs
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+pragma solidity 0.8.22;
+
+import { TrancheWrapper_Fuzz_Test } from "./_TrancheWrapper.fuzz.t.sol";
+
+import { stdStorage, StdStorage } from "../../../lib/accounts-v2/lib/forge-std/src/StdStorage.sol";
+
+/**
+ * @notice Fuzz tests for the function "totalAssets" of contract "Tranche".
+ */
+contract TotalAssets_TrancheWrapper_Fuzz_Test is TrancheWrapper_Fuzz_Test {
+    using stdStorage for StdStorage;
+    /* ///////////////////////////////////////////////////////////////
+                              SETUP
+    /////////////////////////////////////////////////////////////// */
+
+    function setUp() public override {
+        TrancheWrapper_Fuzz_Test.setUp();
+    }
+
+    /*//////////////////////////////////////////////////////////////
+                              TESTS
+    //////////////////////////////////////////////////////////////*/
+    function testFuzz_Success_totalAssets(uint128 assets) public {
+        pool.setRealisedLiquidityOf(address(tranche), assets);
+
+        assertEq(trancheWrapper.totalAssets(), assets);
+
+        //this is redundant, right? totalAssetsAndSync is not a function of the tranche wrapper
```

Comment:
yes indeed, allready tested on tranche

Reply:
Not redundant, since totalAssets() is a valid function on the TrancheWrapper.
Would more call it a trivial test, we just want to ensure that both Tranche and TrancheWrapper always return same value.